src/main/java/com/ddang/usedauction/order/domain/Orders.java

```diff
@@ -0,0 +1,45 @@
+package com.ddang.usedauction.order.domain;
+
+import com.ddang.usedauction.config.BaseTimeEntity;
+import com.ddang.usedauction.member.domain.Member;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.validation.constraints.Min;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+@AllArgsConstructor
+@Builder
+@Entity(name = "orders")
+public class Orders extends BaseTimeEntity {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id; // 주문 아이디
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "member_id", nullable = false)
+    private Member member; // 회원 아이디
+
+    @Column(nullable = false)
+    private String itemName; // 아이템명
+
+    @Column(nullable = false)
+    @Min(value = 1, message = "아이템 가격은 1이상이어야 합니다.")
+    private int price; // 아이템 가격
+
+    @Setter
+    private String tid; // 결제 고유 번호 (결제준비요청 후 받는 응답에 존재. 결제승인처리 로직에 필요. 초기값은 null)
```

Comment:
@Setter 사용하지 않고 메서드로 빼시는건 어떨까요 ??
```suggestion
public void setTid(String tid) {
    this.tid = tid;
}
```

Reply:
저도 원래는 밑에처럼 구현했었는데 위의 방식으로 수정하는 것이 코드가 깔끔해지는 것 같아 것이 위쪽 코드를 선택했습니다!